src/config/dynamic-theme-fixes.config

```
@@ -2872,6 +2872,9 @@ CSS
 html {
     text-shadow: none !important;
 }
+[data-darkreader-inline-fill] {
```

This isn't how a fix is sussposed to be working.

---

may i ask, what is wrong with it. works by me. and is shorter than make it for each element.
else it has to be like this suggestion

```suggestion
#dropdown-control svg {
    fill: ${black} !important;
}
.icon-svg {
```

---

`may i ask, what is wrong with it`
It's overriding a dark reader style CSS(the `--inline` style).

`and is shorter than make it for each element.`
Than let it be for each element as that style shouldn't be override as it will mess up other innocent elements.

---

thanks for response. the suggestion above is edited.